Joseph-DeJesus-Project/src/Max_and_linear_search_algorithm.cpp

```diff
@@ -0,0 +1,62 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+
+// Function to perform Linear Search
+int linearSearch(const vector<int>& arr, int target) {
+    for (int i = 0; i < arr.size(); i++) {
+        if (arr[i] == target) {
+            return i; // return the index of the target
+        }
+    }
+    return -1; // return -1 if the target is not found
+}
+
+// Function to find the maximum element in an array
+int findMax(const vector<int>& arr) {
+    int max = arr[0]; // Assume the first element is the max initially
+    for (int i = 1; i < arr.size(); i++) {
+        if (arr[i] > max) {
+            max = arr[i];
+        }
```

Comment:
I would be good to document what is going on in this `for` loop:
Why are we looping over the length of the array?
What purpose does the `if` statement serve?